llama_stack/distribution/access_control/access_control.py

```diff
@@ -105,5 +105,43 @@ def is_action_allowed(
     return False
 
 
+def build_access_denied_message(action: str | None, resource: ProtectedResource | None, user: User | None) -> str:
```

Comment:
This can go away?

Reply:
sorry, that was a bad merge; having to switch between different branches I am working on.
removing that and force pushing my last commit